perpVects/QFPHelpers.h

```diff
@@ -31,30 +36,30 @@ void printOnce(std::string, void*);
 
 // returns a bitlength equivalent unsigned type for floats
 // and a bitlength equivalent floating type for integral types
-template<typename T>
-struct get_corresponding_type {
-  using type = typename std::conditional_t<
-    std::is_floating_point<T>::value && sizeof(T) == 4, uint32_t,
-    std::conditional_t<
-      std::is_floating_point<T>::value && sizeof(T) == 8, uint64_t,
-    std::conditional_t<
-      std::is_floating_point<T>::value && sizeof(T) == 16, unsigned __int128,
-    std::conditional_t<
-      std::is_integral<T>::value && sizeof(T) == sizeof(float), float,
-    std::conditional_t<
-      std::is_integral<T>::value && sizeof(T) == sizeof(double), double,
-    std::conditional_t<
-      std::is_integral<T>::value && sizeof(T) == sizeof(long double), long double,
-    std::conditional_t<
-      std::is_same<T, __int128>::value && sizeof(long double) == 16, long double,
-    std::conditional_t<
-      std::is_same<T, unsigned __int128>::value && sizeof(long double) == 16, long double,
-      void
-    >>>>>>>>;
-};
-
-template <typename T>
-using get_corresponding_type_t = typename get_corresponding_type<T>::type;
+// template<typename T>
+// struct get_corresponding_type {
```

Comment:
Again, if you don't need this anymore, I suggest deleting it since we have a good version control system.  But not a big deal.